src/modules/helloworld/helloworld.props.autogenerated.ts

```diff
@@ -6,7 +6,7 @@
 
 import * as Msdyn365 from '@msdyn365-commerce/core';
 
-export interface IHelloworldConfig {
+export interface IHelloworldConfig extends Msdyn365.IModuleConfig {
     showText?: string;
```

Comment:
This is an aside, but can we change the variable "showText" to something more realistic like "title", in fact i meant to ask for this and pre-emtively changed docs to show title :-)